src/pages/home/ReportScreen.js

```diff
@@ -127,6 +130,11 @@ class ReportScreen extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        // If a reportActionID exists in the URL then let's close the drawer so that we can see the message comments on native.
+        if (this.getReportActionID()) {
```

Comment:
Shouldn't we only invoke close drawer if the prevProps DID NOT have a reportActionID?

Reply:
> Shouldn't we only invoke close drawer if the prevProps DID NOT have a reportActionID?

atm no because the drawer always reloads opens by default on mobile devices. Hence the logic here is to close the drawer whenever reportActionID is present.

Nice, yes I'll create a followup issue to link to the thread and its issue and once its addressed this can be cleaned up.